api/lib/block-listeners/base-node-subscription.js

```diff
@@ -174,7 +174,8 @@ class BaseNodeSubscription {
       // transaction to each subscribed user.
       // TODO doesn't handle failing txs as it doesn't extract addresses from those txs (they are not tagged)
       block.transactions.forEach((tx) => {
-        tx.involvedAddresses.forEach((address) => {
+        tx.transactionMessageAddresses.forEach((address) => {
+          if (!address) return // case for governance transactions where there is no recipient
```

Comment:
this case should be handled inside of `publishUserTransactionAddedV2`

Reply:
I mean, if we fix involvedAddresses like you propose, we don't need to change anything here at all

Reply:
`involvedAddresses` should never have an item `null`. something is wrong at a different place, if that happens